develop/protoc.sh

```diff
@@ -34,6 +34,14 @@ find "$new" -name service.pb.go -o -name service_grpc.pb.go | while read -r src;
   rm -f "$dst.replaced"
 done
 
+color "Generate external proto mocks..."
+$MOCKGEN -copyright_file LICENSE -package workflowservicemock \
+  -destination "$new"/temporal/api/workflowservicemock/v1/service_grpc.pb.mock.go \
```

Comment:
should it just generate them in
```suggestion
  -destination "$new"/temporal/server/api/workflowservicemock/v1/service_grpc.pb.mock.go \
```
since that's the final location anyway?

Reply:
I was thinking it's not server defined proto so having `server` in the path is a bit weird. But yeah it's just temp dir so I am fine either way.
 
I will update and remove L53 as well.

Reply:
It is weird, but `server` ends up in the import path eventually. Maybe we should put them in `go.temporal.io/server/common/testing/mockapi`, parallel to `mocksdk` (our generated mocks for sdk-go)?

Reply:
hmm `server` is part of the repo's module name so it will always be in the import path.

but yeah `common/testing/mockap` sounds like a better place for them. and I can go back to the go generate approach.